Directory.Packages.props

```diff
@@ -14,13 +14,9 @@
 
   <!-- MSBuild has vulnerable dependencies Microsoft.IO.Redist and System.Security.Cryptography.Xml. When it's upgraded, try removing pinned packages-->
   <PropertyGroup>
-    <!-- The last package version of MSBuild that works with netcoreapp3.1 or netstandard2.0 is 16.8.0.  Our assemblies are still compiled against MSBuild assembly version 15.1.0.0 which will work with all versions -->
-    <MicrosoftBuildVersion Condition="'$(TargetFramework)' == 'netcoreapp3.1' Or '$(TargetFramework)' == 'netstandard2.0'">16.8.0</MicrosoftBuildVersion>
```

Comment:
Since the MSBuild pack targets are not shipped in VS, customers using non-SDK style PackageReference projects who want to pack their projects need to reference the NuGet.Build.Tasks.Pack package. It ships .NET Framework and .NET Standard assemblies. Therefore, by removing the lower MSBuild package version, we're (probably) increasing the minimum VS version that is required to run pack.

Is anyone concerned if customers packing non-SDK style projects can no longer pack with VS 2019 (16.??), or VS 2022 versions 17.0 through 17.10, using the newest version of this package?

Reply:
Interestingly the strong name of MSBuild's assemblies in both package version 16.8.0 and 17.11.x is the same, 15.1.0. So, I don't think there's a risk of runtime assembly load failures after all. Customers referencing the package should be able to run the pack target as far back as VS 15.1.0 (other compat issues aside).